apps/src/netsim/tweens.js

```diff
@@ -0,0 +1,184 @@
+/* jshint
```

Comment:
Our own code.  Josh mentioned we might want to try and use `jQuery.animate()` though so far jQuery's support for SVG isn't stellar.  There are some other library options we could grab too.